src/components/Toast/Toast.stories.tsx

```diff
@@ -18,17 +18,19 @@ export const _Toast = {
         Theme dark doesn't work for Toast, you will see only theme light toats.
       </p>
       <br />
-      <Button
-        onClick={() => {
-          toast.error('Euhh! This is an error message!');
-        }}
-      >
-        Create ERROR Toast
-      </Button>
-      <br />
-      <Button onClick={() => toast.success('Uuurraa! This is a success CSS')}>
-        Create SUCCESS Toast
-      </Button>
+      <div className="theme-dark">
```

Comment:
Why add this "theme-dark" here ? It just makes those 2 buttons white in the storybook🤔
Which is weird as the 3 others buttons are dark

Reply:
in the storybook the toast is present 2 times in the DOM so the last theme win, there is a conflict. I could fix it last time.